Web/ForgeTrust.AppSurface.Docs/wwwroot/css/app.css

```
@@ -1,91 +1,116 @@
+    --docs-color-brand-blue-wash: rgba(37, 99, 235, 0.42);
+    --docs-color-brand-teal-shadow: rgba(20, 184, 166, 0.3);
+    --docs-color-brand-teal-wash: rgba(20, 184, 166, 0.08);
+    --docs-color-brand-violet-shadow: rgba(139, 92, 246, 0.34);
+    --docs-color-brand-violet-wash: rgba(139, 92, 246, 0.1);
+    --docs-color-brand-ice-border: rgba(229, 231, 235, 0.2);
+    --docs-color-brand-ice-highlight: rgba(229, 231, 235, 0.05);
+    --docs-color-brand-panel-border: rgba(65, 87, 121, 0.62);
+    --docs-color-panel-depth-shadow: rgba(0, 0, 0, 0.22);
     --docs-focus-ring-inset: 0 0 0 1px var(--docs-color-accent-strong) inset;
     --docs-focus-outline: 2px solid var(--docs-color-accent-strong);
+    color-scheme: dark;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)